examples/pydantic-ai/provider.py

```
@@ -0,0 +1,44 @@
+"""
+Promptfoo Python provider for PydanticAI agents.
+
+This provider runs PydanticAI agents and returns structured outputs
+for evaluation by promptfoo.
+"""
+
+import asyncio
+import os
+from typing import Any, Dict
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)